src/test/resources/simple/Chart.yaml

```diff
@@ -1,3 +1,3 @@
 apiVersion: v1
 name: app
-version: 0.0.1
+version: 0.1.0
```

Comment:
small change to make this chart's version consistent with the charts tar artifact